compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/JvmLower.kt

```diff
@@ -99,7 +99,7 @@ private val lateinitUsageLoweringPhase = makeIrFilePhase(
     description = "Insert checks for lateinit field references"
 )
 
-private val propertiesPhase = makeIrFilePhase(
+internal val propertiesPhase = makeIrFilePhase(
     ::JvmPropertiesLowering,
     name = "Properties",
     description = "Move fields and accessors for properties to their classes, and create synthetic methods for property annotations",
```

Comment:
Please adapt this phase description to the changes

Reply:
Done.